### ISSUE TITLE
Indent lines for the tag list

### ISSUE THREAD

**Issue by bendem** [fabiospampinato/notable#327]
### Feature description

A picture is worth a thousand words: a small guiding line that allows me to more easily identify the tag hierarchy.
![screenshot 2019-01-23 at 13 35 42](https://user-images.githubusercontent.com/2681677/51607038-e185e900-1f13-11e9-99fb-196d96e2fd29.png)

### Feature motivation

I have many tags and I find it hard to follow which tag has which parent at a quick glance. I'd like some visual help to help me with that.

**Comment by xplatinum** [comment]
I wouldn't mind a thin guiding line but what I would really like is an indent for each level. Right now all notebooks sit directly under the Notebooks header, same with Tags. It hurts my eyes ;)
